src/rgw/rgw_rados.cc

```diff
@@ -1505,6 +1506,7 @@ int RGWZoneParams::fix_pool_names()
   user_email_pool = fix_zone_pool_name(pool_names, name, ".rgw.users.email", user_email_pool.name);
   user_swift_pool = fix_zone_pool_name(pool_names, name, ".rgw.users.swift", user_swift_pool.name);
   user_uid_pool = fix_zone_pool_name(pool_names, name, ".rgw.users.uid", user_uid_pool.name);
+  roles_pool = fix_zone_pool_name(pool_names, name, ".rgw.authorizationdata", roles_pool.name);
```

Comment:
I had a question here - now that we are changing the pool structure to have all the metadata in one pool, will that affect the multisite syncing in any way?